scss/custom/includes/_portfolio.scss

```diff
@@ -6,6 +6,9 @@
 
   .portfolio-upper {
     width: 100%;
+    h1 {
+      font-size: 3rem !important;
```

Comment:
Thank you for the suggestion, Hannah! Both using the `$fontsize-xl` variable and using the `heading1` mixin worked, so I went ahead and used the mixin since it sounds like it would be the better option.

Reply:
You can see these changes on the other PR.